### PR TITLE
restart-capcom timer and service for IT

### PR DESCRIPTION
**Pull request by iderdik**

### CODE REVIEW

**Comment by bryanlatten** [comment]
@iderdik is it necessary to kill it (ungracefully)?

**Comment by iderdik** [comment]
@bryanlatten:  There's no state involved and its a temporary measure until I find the memory leak so I'm doing what is simplest. Suggestions are welcome!

File: v2/fleet-local/it-hybrid/capcom-restart.service [hunk 1]
```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=Capcom Restart
+After=docker.service
+Requires=docker.service
+
+[Service]
+User=core
+RestartSec=20
+TimeoutStartSec=0
+ExecStart=/usr/bin/sudo docker kill capcom
```
Comment (bryanlatten) [inline v2/fleet-local/it-hybrid/capcom-restart.service]: `docker stop capcom`

**Comment by bryanlatten** [comment]
@iderdik let's quickly add some comments about why we're doing this, and that its a temporary measure, then 👍

**Comment by nBerg** [comment]
:+1:

**Comment by chr0n1x** [comment]
👍

**Pull request merged**